File: dev/ClientApp/src/app/models/iranKish/LinkRequest.ts
import FactorDetail from '../FactorDetail.Model';

export default class LinkRequest {
  constructor(
    public terminalId: number,
    public MID: number,
    public State: string,
    public InsDate: string,

    public Status: number,

    public title: string,

    public price: number,

    public orderId: string,

    public factorDetailJson: string

  ) { }
}

File: Models/LinkRequest.cs
﻿using System;
using System.Collections.Generic;
using NodaTime;

namespace G_IPG_API.Models;

public partial class LinkRequest
{
    public long? UserId { get; set; }
    public long? WalletId { get; set; }
    public long? WallectCurrencyId { get; set; }
    public long? TransactionConfirmId { get; set; }
    public string? Guid { get; set; }

    public int RequestId { get; set; }

    public DateTime InsertDate { get; set; }

    public short Status { get; set; }

    public string Title { get; set; } = null!;

    public decimal? Price { get; set; }

    public short CallBackType { get; set; }

    public string? OrderId { get; set; }

    public DateTime ExpireDate { get; set; }

    public string? AccLinkReqConf { get; set; }

    public string? CallbackUrl { get; set; }

    public string? ClientMobile { get; set; }

    public string? FactorDetail { get; set; }
}
